lib/Mojolicious/Guides/Rendering.pod

```diff
@@ -443,8 +443,8 @@ L<Mojolicious::Plugin::DefaultHelpers/"accepts">.
 
 By now you've probably already encountered the built-in 404 (Not Found) and 500
 (Server Error) pages, that get rendered automatically when you make a mistake.
-Those are fallbacks for when your own exception handling fails, but especially
-during development they can also be a great help, you can render them manually
+Those are fallbacks for when your own exception handling fails. Especially
+during development, they can also be a great help: you can render them manually
```

Comment:
Propose a third alternative

```
Those are fallbacks for when your own exception handling fails, which can be especially
helpful during development. You can also render them manually ...
```

that change would require rebalancing the lines of course